doc: Move Getting started with nRF52 Series to NCS from infocenter (#8338)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf52_features.rst

.. _ug_nrf52_features:

Features of nRF52 Series
########################

.. contents::
   :local:
   :depth: 2

> The nRF52 Series of System-on-Chip (SoC) devices embed a powerful yet low-power Arm Cortex-M4 processor with Nordic Semiconductor's 2.4 GHz RF transceivers.

Reviewer: ```suggestion
The nRF52 Series of System-on-Chip (SoC) devices embed an Arm Cortex-M4 processor with Nordic Semiconductor's 2.4 GHz RF transceivers.
```
Marketing words?

Author: Good catch.

All of the nRF52 Series SoCs have support for Bluetooth® 5 features, in addition to multiprotocol capabilities.

For additional information, see the following documentation:

* `nRF52 Series`_ for the technical documentation on the nRF52 Series chips and associated kits.
* :ref:`ug_nrf52_gs`.
* :ref:`ug_nrf52` for more advanced topics related to the nRF52 Series.
* The :ref:`introductory documentation <getting_started>` if you are not familiar with the |NCS| and the development environment.

.. _nrf52_supported_boards:

Supported boards
****************

Devices in the nRF52 Series are supported by these boards in the Zephyr open source project and in |NCS|.

> 

Author: Fixed.

       | `User Guide <nRF52 DK User Guide_>`_
   * - :ref:`zephyr:nrf52dk_nrf52810`
     - PCA10040
     - ``nrf52dk_nrf52810``
     - `Product Specification <nRF52810 Product Specification_>`_
   * - :ref:`zephyr:nrf52dk_nrf52805`
     - PCA10040
     - ``nrf52dk_nrf52805``
     - `Product Specification <nRF52805 Product Specification_>`_
   * - :ref:`zephyr:nrf52840dongle_nrf52840`
     - PCA10059
     - ``nrf52840dongle_nrf52840``
     - | `Product Specification <nRF52840 Product Specification_>`_
       | `User Guide <nRF52840 Dongle User Guide_>`_
   * - :ref:`zephyr:nrf21540dk_nrf52840`
     - PCA10112
     - ``nrf21540dk_nrf52840``
     - `Product Specification <nRF21540 Product Specification_>`_

See :ref:`ug_radio_fem_nrf21540_ek` to learn how to use this RF front-end module (FEM) with the nRF52 Series devices.

Secure bootloader chain
***********************

nRF52 Series devices support a secure bootloader solution based on the chain of trust concept.

See :ref:`ug_bootloader` for more information and instructions on how to enable one or more bootloaders in your application.

Supported protocols
*******************

> The nRF52 Series multiprotocol radio supports Bluetooth Low Energy, proprietary (including Enhanced ShockBurst and Gazell), ANT, Thread, Zigbee, and 802.15.4.

Author: Fixed.

> The nRF52 Series multiprotocol radio supports Bluetooth Low Energy (LE), proprietary (including Enhanced ShockBurst and Gazell), ANT, Thread, Zigbee, and 802.15.4.

Reviewer: ```suggestion
The nRF52 Series multiprotocol radio supports Bluetooth Low Energy (LE), proprietary protocols (including Enhanced ShockBurst and Gazell), ANT, Thread, Zigbee, and 802.15.4.
```

Author: Fixed.

Standard interface protocols like NFC and USB are supported on a range of the devices in the series and with supporting software.

.. note::
   |NCS| currently has no support for ANT.

The following sections give pointers on where to start when working with these protocols.

To test the general capabilities of the 2.4 GHz radio transceiver, use the :ref:`radio_test` sample.

Bluetooth Low Energy
====================

> When you develop a Bluetooth Low Energy (LE) application, you must use the Bluetooth software stack.

Author: Fixed.

This stack is split into two core components: the Bluetooth Host and the Bluetooth LE Controller.

The |NCS| Bluetooth stack is fully supported by Nordic Semiconductor for nRF52 Series devices.
The :ref:`ug_ble_controller` user guide contains more information about the two available Bluetooth LE Controllers and instructions for switching between them.

> See :ref:`zephyr:bluetooth` for documentation on the Bluetooth Host and open source LE Controller.

Author: Fixed.

For documentation about the SoftDevice Controller and information on which controller variants support which chips, see :ref:`nrfxlib:softdevice_controller`.

The |NCS| contains a variety of :ref:`ble_samples` that target nRF52 Series devices.
In addition, you can run the :ref:`zephyr:bluetooth-samples` that are included from Zephyr.

For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).

Bluetooth mesh
==============

> Bluetooth mesh is a mesh networking solution based on Bluetooth Low Energy, developed by the Bluetooth Special Interest Group (SIG).

Author: Fixed.

It is optimized for creating large-scale device networks, and implemented according to Bluetooth Mesh Profile Specification v1.0.1 and Bluetooth Mesh Model Specification v1.0.1.

> Bluetooth mesh networking allows one-to-one, one-to-many, and many-to-many communication, using the Bluetooth Low Energy protocol to exchange messages between the mesh nodes in the network.

Author: Fixed.
